### ISSUE TITLE
add cloudresourcemanager.googleapis.com api to shared_vpc example

### ISSUE THREAD

**Issue by apeabody** [terraform-google-modules/terraform-google-project-factory#717]
### TL;DR

add cloudresourcemanager.googleapis.com api to shared_vpc example

### Observed behavior

Error: Error when reading or editing Resource [...] for IAM Member (role "serviceAccount:service-[...]@container-engine-robot.iam.gserviceaccount.com", "roles/container.hostServiceAgentUser"): Request `Delete IAM Members roles/container.hostServiceAgentUser serviceAccount:service-[...]@container-engine-robot.iam.gserviceaccount.com for "project \"[...]\""` returned error: Error retrieving IAM policy for project "[...]": googleapi: Error 403: Cloud Resource Manager API has not been used in project [...] before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview?project=[...] then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.

### Terraform Configuration

```hcl
CI
```

### Terraform Version

```sh
CI
```
